src/Sarif.Sdk.Sample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Consider placing this whole thing into a `samples` directory. Since we are likely to have more than one sample, consider placing this one into a subdirectory. That would require you to come up with a name more specific than `Sdk.Sample`, which I think would be a good thing.

Having said all that, if your goal here is just to get this checked in ASAP, then at least put it as-is under a `src\samples` directory -- we can rename the sample, and subdivide the directory, later, if needed.

Reply:
Ok, looking at what the sample does, how about `Sarif.Sdk.ReadWriteSample`, in directory `src\samples\readWrite`?

Reply:
Done